mission_yaml/SEA55_M23.yml

```diff
@@ -0,0 +1,456 @@
+metadata:
+  acknowledgement: This study used data collected and made freely available by Voice of the Ocean Foundation
+                   (https://www.voiceoftheocean.org) accessed from https://erddap.voiceoftheocean.org
+  institution: Voice of the Ocean Foundation
+  license: Creative Commons Attribution 4.0 (https://creativecommons.org/licenses/by/4.0/)
+    This study used data collected and made freely available by Voice of the Ocean Foundation
+    (https://www.voiceoftheocean.org) accessed from https://erddap.voiceoftheocean.org
+  format_version: IOOS_Glider_NetCDF_v2.0.nc
+  glider_model: SeaExplorer
+  glider_instrument_name: seaexplorer
+  keywords: "AUVS, Autonomous Underwater Vehicles, Oceans, Ocean Pressure,
+             Water Pressure, Ocean Temperature, Water Temperature,
+             Salinity/Density, Conductivity, Density, Salinity"
+  keywords_vocabulary: GCMD Science Keywords
+  metadata_link: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
+  Metadata_Conventions: CF-1.6, Unidata Dataset Discovery v1.0
+  naming_authority: Voice of the Ocean Foundation
+  platform:    "SeaExplorer Glider"
+  processing_level: "Data provided as is with no expressed or implied
+                     assurance of quality assurance or quality control. L0"
+  publisher_email: callum.rollo@voiceoftheocean.org
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  transmission_system: IRIDIUM
+  glider_name: Kaprifol
+  glider_serial: "55"
+  wmo_id: "6801674"
+  comment: 'deployment and recovery in Gotland'
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://callumrollo.com/
+  deployment_id: "23"
+  deployment_name: AUV_project
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2021-08-21"
+  deployment_end: "2021-09-09"
+  project: AUV project
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of AUV project'
+  # Examples: Skagerrak, Kattegatt, Baltic, Gulf of Bothnia, Gulf of Finland, Gulf of Riga
+  sea_name: Gotland
```

Comment:
sea_name need to be something different. Probably "Baltic" as well. If we want to be specific we could change it to "Gotland Basin" or "Eastern Gotland Basin", but then we would nee to be specific in Bornholm as well, setting the sea_name to "Southern baltic proper" or similar.

Reply:
I'd leave it as Baltic for now, if we decide to change it later we could trigger reprocessing then

Reply:
I agree. It should be Baltic